chapter_6/ex01_doxdocgen/CMakeLists.txt

```diff
@@ -0,0 +1,94 @@
+# CMakeLists file for the Chapter 6 Example nr. 01: 
+# Doxygen documentation generation
+#
+# SPDX-License-Identifier: MIT
+
+project(
+  ch6_ex01_doxdocgen
+  VERSION 1.0
+  DESCRIPTION "Chapter 6 Example 01, Doxygen documentation generation"
+  LANGUAGES CXX)
+
+############################################################################
+# Doxygen configuration
+############################################################################
+# Locate Doxygen in the environment. This will set DOXYGEN_FOUND variable 
+# as a result. When Doxygen is found, DOXYGEN_VERSION variable will be 
+# also set to found Doxygen version.
+find_package(Doxygen REQUIRED)
```

Comment:
For the sake of accessibility of the code I would not make this `REQUIRED` and surround the relevant part of the file with and `if(DOXYGEN_FOUND)`

This will allow people to build the whole project even if they miss oxygen.

Reply:
Yes, you are right. It would be better that way. I'll change it accordingly.